pkg/pullsync/metrics.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package pullsync
+
+import (
+	m "github.com/ethersphere/bee/pkg/metrics"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type metrics struct {
+	OfferCounter    prometheus.Counter // number of chunks offered
+	WantCounter     prometheus.Counter // number of chunks wanted
+	DeliveryCounter prometheus.Counter // number of chunk deliveries
+	DbOpsCounter    prometheus.Counter // number of db ops
+}
+
+func newMetrics() metrics {
+	subsystem := "pullsync"
+
+	return metrics{
+		OfferCounter: prometheus.NewCounter(prometheus.CounterOpts{
+			Namespace: m.Namespace,
+			Subsystem: subsystem,
+			Name:      "chunks_offered",
+			Help:      "Total chunks offered.",
+		}),
+		WantCounter: prometheus.NewCounter(prometheus.CounterOpts{
+			Namespace: m.Namespace,
+			Subsystem: subsystem,
+			Name:      "chunks_wanted",
+			Help:      "Total chunks wanted.",
+		}),
+		DeliveryCounter: prometheus.NewCounter(prometheus.CounterOpts{
+			Namespace: m.Namespace,
+			Subsystem: subsystem,
+			Name:      "chunks_delivered",
+			Help:      "Total chunks delivered.",
+		}),
```

Comment:
same here, this means delivered _to us_, no?

Reply:
yeah, and that's where it is incremented. where's the question?

Reply:
no question, i would just make it more explicit in the `Help` text (_delivered to whom?_)

not a blocker though, and i'm happy to let it go if it's just me :)